tools/notification-configuration/notification-creator/NotificationConfigurator.cs

```diff
@@ -152,9 +165,85 @@ public NotificationConfigurator(AzureDevOpsService service, ILogger<Notification
                 }
```

Comment:
We should also add the update the subscriptions here when we are updating this.

Reply:
Done.